src/components/PokemonList.jsx

```diff
@@ -0,0 +1,33 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+import Pokemon from './Pokemon';
+import './css/PokemonList.scss';
+
+const PokemonList = () => {
+  /* Declaration of a variable with useState to store the return value of the
+    call to the API (useSate initialized with an empty array) */
+  const [pokemons, setPokemons] = useState([]);
+
+  /* Call the API to get the list of pokémons that we want to display on the
+    Homepage (here the 50 first Pokemon) */
+  useEffect(() => {
+    axios
+      .get('https://pokeapi.co/api/v2/pokemon?limit=20&offset=0')
```

Comment:
The comment says 50 Pokemon ; the code shows it's limited to the first 20 results